api/src/main/java/io/opentelemetry/trace/NoopTrace.java

```diff
@@ -121,6 +105,21 @@ public Span startSpan() {
       return BlankSpan.INSTANCE;
     }
 
+    @Override
+    public NoopSpanBuilder setParent(@Nullable Span parent) {
```

Comment:
this should not accept null anymore.

Reply:
Agreed. And to be clear: now users *should* use the `ignoreCurrentSpan()` method (or `newTrace()` or whatever name we give to it). Makes sense?

Reply:
Yes.